packages/fast-usdc/src/cli/bridge-action.js

```diff
@@ -25,8 +28,13 @@ export const sendHint =
  *   stdout: Pick<import('stream').Writable,'write'>,
  *   stderr: Pick<import('stream').Writable,'write'>,
  * }} io
+ * @param {BoardSlottingMarshaller | undefined} marshaller
  */
-export const outputActionAndHint = (bridgeAction, { stdout, stderr }) => {
-  outputAction(bridgeAction, stdout);
+export const outputActionAndHint = (
+  bridgeAction,
+  { stdout, stderr },
+  marshaller = defaultMarshaller,
+) => {
+  outputAction(bridgeAction, stdout, marshaller);
```

Comment:
:thinking: 
I don't understand how that typechecks. `undefined` isn't a `BoardSlottingMarshaller`